keyboards/annepro2/c18/rules.mk

```diff
@@ -36,7 +36,7 @@ EXTRAKEY_ENABLE = yes
 KEY_LOCK_ENABLE = no
 
 # Other featues
-BOOTMAGIC_ENABLE = no
+BOOTMAGIC_ENABLE = yes
```

Comment:
Pretty sure you don't need to enable this but it's fine

Reply:
Right, `COMMAND_ENABLE = yes` on line 41 handles it. There is some overlap between shortcuts. On my local fork I had `COMMAND_ENABLE` disabled and that's why suggested `BOOTMAGIC_ENABLE` change.